src/kernel/include/shared/consts.h

```diff
@@ -106,6 +106,8 @@ struct pt_regs {
 
 #define COS_NUM_ATOMIC_SECTIONS 10
 
+#define TCAP_MAX 32
```

Comment:
Don't think this is used anywhere.  If it is, then it shouldn't exist.  Burn it!!!